src/common/AndroidUtils.ts

```diff
@@ -464,7 +464,7 @@ export class AndroidSDKUtils {
                 AndroidSDKUtils.logger.info(
                     `Installing app ${appBundlePath.trim()} to emulator`
                 );
-                const installCommand = `${AndroidSDKUtils.getAdbShellCommand()} -s emulator-${emulatorPort} install -r -t '${appBundlePath.trim()}'`;
+                const installCommand = `${AndroidSDKUtils.getAdbShellCommand()} -s emulator-${emulatorPort} install -r -t "${appBundlePath.trim()}"`;
```

Comment:
This doesn't seem like a change you'd want. Special characters in the path could be misinterpreted without the single quotes.

Reply:
I made this change b/c on Windows ADB wasn't working with single quote but it was working with double quotes. I forgot about special characters on iOS. I'll change this to use double quotes only on Windows.

Reply:
Oh that makes sense. I once again forgot about Windows. 🙄